src/components/headers/BasicHeader.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+import { ITitleProps, Title } from '../title/Title';
+import { HeaderWrapper, IHeaderWrapperProps } from './HeaderWrapper';
+
+export interface IBasicHeaderProps extends IHeaderWrapperProps, React.ClassAttributes<React.Component<any, any>> {
+  title: ITitleProps;
+}
+
+export class BasicHeader extends React.Component<IBasicHeaderProps, {}> {
+
+  static defaultProps: Partial<IBasicHeaderProps> = {
+    title: {
+      title: '',
```

Comment:
title inside of title, that seems a little weird, maybe we can change the name of one of them

Reply:
`text` ?